source/stdx/allocator/building_blocks/segregator.d

```diff
@@ -247,7 +247,7 @@ struct Segregator(size_t threshold, SmallAllocator, LargeAllocator)
                 && __traits(hasMember, LargeAllocator, "empty"))
         Ternary empty()
         {
-            return _small.empty && _large.empty;
+            return _small.empty & _large.empty;
```

Comment:
nice, is the phobos version affected too ?

Reply:
was already fixed in https://github.com/dlang/phobos/commit/59c25d8cc8654ca4dcc6e8f029dc08ef1f51446e in 2017

Reply:
Thanks for the info. I need to back port this to the 2.77.z branch however